src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo | null,
+  selectedTodo: React.Dispatch<React.SetStateAction<number>>
+};
+
+export const TodoModal: React.FC<Props> = ({
+  todo,
+  selectedTodo,
+}) => {
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    if (todo) {
+      getUser(todo.userId)
+        .then(findedUser => setUser(findedUser));
+    }
+  }, []);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {(!user || !todo) ? (
```

Comment:
same here about loader, will be better to create separate variable for handling was data loaded or no